views/templates/admin/tabs/help.tpl

```diff
@@ -0,0 +1,94 @@
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2018 PrestaShop SA
+* @license   http://addons.prestashop.com/en/content/12-terms-and-conditions-of-use
+* International Registered Trademark & Property of PrestaShop SA
+*}
+
+<div class="panel col-lg-10 right-panel">
+    <h3>
+        <i class="fa fa-question-circle"></i> {l s='Help & contact' d='Modules.Faviconnotificationbo.Admin'} <small>{$module_display|escape:'htmlall':'UTF-8'}</small>
+    </h3>
+    <div class="helpContentParent">
+        <div class="helpContentLeft">
+            <div class="left">
+                <img src="{$logo_path|escape:'htmlall':'UTF-8'}" alt=""/>
+            </div>
+            <div class="right">
+                <p><span class="data_label" style="color:#00aff0;"><b>{l s='This module allows you to :' d='Modules.Faviconnotificationbo.Admin'}</b></span></p>
+                <br>
+                <div>
+                    <div class="numberCircle">1</div>
+                    <div class="numberCircleText">
+                    <p class="numberCircleText">
+                        {l s='Erase any customer account with his/her personal data collected by your shop if requested by the customer' d='Modules.Faviconnotificationbo.Admin'}
+                    </p>
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">2</div>
+                    <div class="numberCircleText">
+                    <p class="numberCircleText">
+                        {l s='Add a consent confirmation checkbox in a module form that collects personal data and customize it' d='Modules.Faviconnotificationbo.Admin'}
+                    </p>
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">3</div>
+                    <div class="numberCircleText">
+                    <p class="numberCircleText">
+                        {l s='Allow your customer to consult and export their personal data collected by your shop on their customer account' d='Modules.Faviconnotificationbo.Admin'}
+                    </p>
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">4</div>
+                    <div class="numberCircleText">
+                    <p class="numberCircleText">
+                        {l s='View all your customers’ actions related to their personal data' d='Modules.Faviconnotificationbo.Admin'}
+                    </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="helpContentRight">
+            <div class="helpContentRight-sub">
+                <b>{l s='Need help ?' d='Modules.Faviconnotificationbo.Admin'}</b><br>
+                {l s='Find here the documentation of this module' d='Modules.Faviconnotificationbo.Admin'}
```

Comment:
I would use a more friendly tone here: _Have a look at the documentation_ (_Admin.Global_) or _Go to the documentation_ (_Admin.Navigation.Search_).